webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: 'cheap-module-eval-source-map',
+  entry: [
+    'webpack-hot-middleware/client',
+    './app'
+  ],
+  output: {
+    path: path.join(__dirname, 'bundle'),
+    filename: 'app.js',
+    publicPath: '/bundle/'
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
```

Comment:
Why not make the `preferEntry` true? I haven't used this plugin, so I'm not what the effects will be, but the document recommends to do so.
[occurenceorderplugin](http://webpack.github.io/docs/list-of-plugins.html#occurenceorderplugin)